psi4/src/psi4/libmoinfo/moinfo_scf.h

```diff
@@ -51,7 +48,7 @@ class MOInfoSCF : public MOInfoBase {
     void print_mo();
 };
 
-extern MOInfoSCF* moinfo_scf;
+extern MOInfoSCF* moinfo_scf;  // Unfortunate, but psi::mcscf assumes that this is declared.
```

Comment:
Thanks for leaving the note. Ideally this would not be a global variable. But I am not sure it's worth the effort to fix this.

Reply:
My thoughts exactly. Briefly looked into it, seemed painful to change.